pkgs/applications/networking/browsers/next/default.nix

```diff
@@ -0,0 +1,106 @@
+{ pkgs, stdenv, fetchFromGitHub
+, gcc7, pkg-config, makeWrapper
+, glib-networking, webkitgtk, gsettings-desktop-schemas
+, lispPackages
+, sbcl
+}:
+
+let
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "atlas-engineer";
+    repo = "next";
+    rev = version;
+    sha256 = "01fn1f080ydk0wj1bwkyakqz93bdq9xb5x8qz820jpl9id17bqgj";
+  };
+
+  next-gtk-webkit = stdenv.mkDerivation rec {
```

Comment:
i feel like this should be broken out into it's own file/package. you could still do:
```
inherit (next) src version;
```